custom_components/healthchecksio/config_flow.py

```
@@ -109,34 +110,111 @@ def _clean_url(url: str) -> str:
     return urlunparse(cleaned)
 
 
+def _build_user_input_schema(
```

Pylint (3.3.7)

[warning] Dangerous default value {} as argument

(W0102)